### PR TITLE
fix: false message about no comments and btnTop scroll

### PR DESCRIPTION
**Pull request by nik-conder**
In new versions (not yet released), comments are broken. Including responses to comments. 
I tried to fix btn Top and corrected the false text output that there are no comments when in fact there are. I haven't figured out the reason yet, so the code may need to be adjusted or supplemented.

### CODE REVIEW

File: app/src/main/java/com/github/libretube/ui/fragments/CommentsRepliesFragment.kt [hunk 1]
```diff
@@ -88,8 +119,18 @@ class CommentsRepliesFragment : Fragment() {
...
+        scrollListener = null
+    }
+
     override fun onDestroyView() {
         super.onDestroyView()
         _binding = null
     }
+
+    companion object {
+        const val POSITION_START = 0
```
Comment (Isira-Seneviratne) [inline app/src/main/java/com/github/libretube/ui/fragments/CommentsRepliesFragment.kt:134]:
```suggestion
        private const val POSITION_START = 0
```

**Review by Isira-Seneviratne — CHANGES_REQUESTED** [review]
Looks good to me overall, thanks for the comment fix. I've suggested a couple of small changes.

**Comment by nik-conder** [comment]
Sorry, the following commit accidentally flew in here, which does not relate to the previous fix. Isn't that scary?

**Comment by Bnyro** [comment]
Merged in https://github.com/libre-tube/LibreTube/pull/5751, thanks!

Please don't merge the LibreTube master branch into yours next time and don't disable that others can push your branch.

**Comment by nik-conder** [comment]
@Bnyro Thank you very much. Sorry again, I'm just learning.

> Please don't merge the LibreTube master branch into yours next time

Please specify what is meant here?

**Pull request closed without merging**